<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterPwdPromptDialog</class>
 <widget class="QDialog" name="MasterPwdPromptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wymagane uwierzytelnienie</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>170</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="icon_pwdIcon">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>14</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/icons/about.svg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_masterPwdLabel">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Podaj hasło główne:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_masterPwd">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>18</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_togglePwdVisibility">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>18</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pokaż hasło</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/eye-solid.svg</normaloff>:/resources/icons/eye-solid.svg</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MasterPwdPromptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MasterPwdPromptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
